packages/cli/src/commands/ui/create/vue.ts

```diff
@@ -182,7 +186,7 @@ export default class Vue extends Command {
     npm run start
 
     See package.json for other available commands.
-    Happy hacking !
+    Happy hacking!
```

Comment:
This one is a bit special, we use it in our E2E test to detect when the command completes.

You can change it, no prob, but make sure to replace the regexp in `packages\cli-e2e\__tests__\vue.specs.ts` L92